drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -168,6 +169,7 @@ enum wifi_nrf_status umac_cmd_init(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 	umac_cmd_data->ant_gain_5g_band1 = ant_gain_5g_band1;
 	umac_cmd_data->ant_gain_5g_band2 = ant_gain_5g_band2;
 	umac_cmd_data->ant_gain_5g_band3 = ant_gain_5g_band3;
+	umac_cmd_data->single_band_only = single_band;
```

Comment:
The input arg should also be named `single_band_only` and also, the naming is not quite right, we should instead use an enum `supported_bands` with below or at the very least rename to `band_24g_only` (my preference is enum).

```
BAND_24G
BAND_5G
BAND_ALL
```

Reply:
I guess we can have a enum, but not have the BANG_5G value in it right now (since we don't support 5 GHz only operation yet).

Reply:
yeah, we could keep it generic and UMAC can fail if a combination isn't supported either way is fine with me.

Reply:
Considering that on UMAC side, this variable is used just to identify if single band support is enabled or not (umac_cmd_data->single_band_only), I think we should define the enum as:
```
enum single_band_24g {
Enabled 
Disabled
}
```

Reply:
As UMAC patch is in this PR, we should change it to take enum as well.

Reply:
Done